c/meterpreter/source/extensions/priv/namedpipe_efs.c

```diff
@@ -24,12 +26,10 @@ DWORD elevate_via_namedpipe_efs(Remote* remote, Packet* packet)
 	WCHAR cPipeName2[MAX_PATH] = { 0 };
 	DWORD dwPipeUid[2] = { 0, 0 };
 	PRIV_POST_IMPERSONATION PostImpersonation;
+	const RPC_WSTR *wEndpointUUID = NULL;
+	const RPC_WSTR *wNamedPipeEndpoint = NULL;
```

Comment:
Why not just have these be `RPC_WSTR` instead of  `RPC_WSTR *`? Seems like you aren't using them for anything but their RPC_WSTR versions and aren't passing them into any calls that expect an `RPC_WSTR *` argument.

Reply:
using direct data type allocates more memory. by pointers we just keep addresses of previous allocated memory.

Reply:
Makes sense, will keep this as is then.